itest/litd_mode_integrated_test.go

```diff
@@ -416,7 +418,7 @@ func integratedTestSuite(ctx context.Context, net *NetworkHarness, t *testing.T,
 					endpoint.macaroonFn(cfg),
 					endpoint.requestFn,
 					endpoint.successPattern,
-					endpointDisabled,
+					endpointDisabled || endpoint.litOnly,
```

Comment:
Shouldn't this have exactly the opposite meaning as in the proxy server should never be disabled? So we should never get the `disabledErr` returned?

Reply:
Ah okay, makes sense, thanks.